.github/workflows/auto-build.yml

```diff
@@ -4,8 +4,10 @@ on:
   push:
     branches:
       - 'master'
-    paths:
-      - 'gradle.properties'
+  pull_request:
+    types:
+      - labeled
+      - synchronize
 
```

Comment:
remove path is fine. but building for all `pull_request` will cause too much actions I think?

Reply:
I tried to make it only work for tagged PRs
Guess it didn't :shrug:

Reply:
Sorry, i meant labeled as `Deployment: artifact`